src/main/java/de/hysky/skyblocker/skyblock/shortcut/ShortcutsConfigListWidget.java

```diff
@@ -8,267 +8,329 @@
+			this(shortcutsMap, targetName, replacementName, Text.translatable(tooltip));
+		}
+
+		private ShortcutCategoryEntry(Map<String, String> shortcutsMap, @Translatable String targetName, @Translatable String replacementName, @Nullable Text tooltip) {
+			this.shortcutsMap = shortcutsMap;
+			this.targetName = Text.translatable(targetName);
+			this.replacementName = Text.translatable(replacementName);
+			this.tooltip = tooltip;
+			shortcutMaps.add(shortcutsMap);
+			addEntry(this);
+		}
+
+		@Override
+		public List<? extends Element> children() {
+			return List.of();
+		}
+
+		@Override
+		public List<? extends Selectable> selectableChildren() {
+			return List.of(new Selectable() {
+				@Override
+				public SelectionType getType() {
+					return SelectionType.HOVERED;
+				}
+
+				@Override
+				public void appendNarrations(NarrationMessageBuilder builder) {
+					builder.put(NarrationPart.TITLE, targetName, replacementName);
+				}
+			});
+		}
+
+		@Override
+		public void render(DrawContext context, int index, int y, int x, int entryWidth, int entryHeight, int mouseX, int mouseY, boolean hovered, float tickDelta) {
+			context.drawCenteredTextWithShadow(client.textRenderer, targetName, width / 2 - 85, y + 5, 0xFFFFFF);
+			context.drawCenteredTextWithShadow(client.textRenderer, replacementName, width / 2 + 85, y + 5, 0xFFFFFF);
+			context.drawCenteredTextWithShadow(client.textRenderer, bindName, width / 2 + 180, y + 5, 0xFFFFFF);
+			if (tooltip != null && isMouseOver(mouseX, mouseY)) {
+				screen.setTooltip(tooltip);
+			}
+		}
+
+		/**
+		 * Returns true so that category entries can be focused and selected, so that we can add shortcut entries after them.
+		 */
+		@Override
+		public boolean mouseClicked(double mouseX, double mouseY, int button) {
+			return true;
+		}
+	}
+
+	private class ShortcutLoadingEntry extends AbstractShortcutEntry {
+		private final Text text;
+
+		private ShortcutLoadingEntry() {
+			this.text = Text.translatable("skyblocker.shortcuts.notLoaded");
+		}
+
+		@Override
+		public List<? extends Element> children() {
+			return List.of();
+		}
+
+		@Override
+		public List<? extends Selectable> selectableChildren() {
+			return List.of(new Selectable() {
+				@Override
+				public SelectionType getType() {
+					return SelectionType.HOVERED;
+				}
+
+				@Override
+				public void appendNarrations(NarrationMessageBuilder builder) {
+					builder.put(NarrationPart.TITLE, text);
+				}
+			});
+		}
+
+		@Override
+		public void render(DrawContext context, int index, int y, int x, int entryWidth, int entryHeight, int mouseX, int mouseY, boolean hovered, float tickDelta) {
+			context.drawCenteredTextWithShadow(client.textRenderer, text, width / 2, y + 5, 0xFFFFFF);
+		}
+	}
+
+	protected class ShortcutEntry extends AbstractShortcutEntry {
+		private final List<Element> children;
+		private final List<Selectable> selectable;
```

Comment:
This should do, no need for two lists.

```suggestion
		private final List<ClickableWidget> children;
```